.github/scripts/test_tryrebase.py

```diff
@@ -5,13 +5,15 @@
 from typing import Any
 from tryrebase import rebase_onto
 
-
+def mocked_rev_parse(branch: str) -> str:
+    return branch
 class TestRebase(TestCase):
 
     @mock.patch('trymerge.gh_graphql', side_effect=mocked_gh_graphql)
     @mock.patch('gitutils.GitRepo._run_git')
+    @mock.patch('gitutils.GitRepo.rev_parse', side_effect=mocked_rev_parse)
     @mock.patch('tryrebase.gh_post_comment')
-    def test_rebase(self, mocked_post_comment: Any, mocked_run_git: Any, mocked_gql: Any) -> None:
+    def test_rebase(self, mocked_post_comment: Any, mocked_rp: Any, mocked_run_git: Any, mocked_gql: Any) -> None:
```

Comment:
what do you think about adding a test that validates the new exception gets thrown?

Reply:
done